### PR TITLE
chore: Linter issues

### PR DESCRIPTION
**Pull request by enemycnt**
## Related Issue Or Context

Closes: #129

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: packages/sdk/integration/test/utils.browser.test.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```
Comment (BeroBurny) [inline packages/sdk/integration/test/utils.browser.test.ts:1]: you should not use this

File: packages/sdk/src/Sygma.ts [hunk 2]
```diff
@@ -148,6 +151,7 @@ export class Sygma implements SygmaSDK {
     }
 
     if (!chain1) {
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```
Comment (BeroBurny) [inline packages/sdk/src/Sygma.ts:154]: it is unreasonable to use `eslint-disable-next-line` on this one, please look at the issue and solve it in the correct way.

File: packages/sdk/src/fee/__test__/feeOracle.test.ts [hunk 3]
```diff
@@ -100,20 +106,20 @@ describe('feeOracle', () => {
         feeOracleBaseUrl: 'http://localhost:8091',
         feeOracleHandlerAddress: '0xa9ddD97e1762920679f3C20ec779D79a81903c0B',
         overridedResourceId: '0xbA2aE424d960c26247Dd6c32edC70B295c744C43',
-        oraclePrivateKey: '0x6937d1d0b52f2fa7f4e071c7e64934ad988a8f21c6bf4f323fc19af4c77e3c5e'
+        oraclePrivateKey: '0x6937d1d0b52f2fa7f4e071c7e64934ad988a8f21c6bf4f323fc19af4c77e3c5e',
       });
-      console.log("🚀 ~ file: feeOracle.test.ts ~ line 99 ~ it ~ feeData", feeData)
+      console.log('🚀 ~ file: feeOracle.test.ts ~ line 99 ~ it ~ feeData', feeData);
```
Comment (BeroBurny) [inline packages/sdk/src/fee/__test__/feeOracle.test.ts:111]: this statement is false, as is currently on line `111`

File: packages/sdk/.eslintrc.js [hunk 4]
```diff
@@ -1,3 +1,7 @@
 module.exports = {
   extends: "../../.eslintrc.js",
+  parserOptions: {
+    project: './tsconfig.json',
+    tsconfigRootDir: __dirname,
+  },
```
Comment (BeroBurny) [inline packages/sdk/.eslintrc.js:6]: is there a reason why is `parserOptions` overridden for an SDK?

File: packages/sdk/src/fee/feeOracle.ts [hunk 5]
```diff
@@ -92,7 +94,8 @@ export const calculateFeeData = async ({
       toDomainID,
       resourceID: overridedResourceId ? overridedResourceId : resourceID, // dirty hack for localsetup
     });
-  } catch (e: any) {
+  } catch (e) {
+    //@ts-ignore-line
```
Comment (BeroBurny) [inline packages/sdk/src/fee/feeOracle.ts:98]: this comment this already

**Review by BeroBurny — COMMENTED** [review]
It is bad practice to use ignore flags to avoid issues

@P1sar why I'm not set as a reviver on this PR?

**Review by BeroBurny — COMMENTED** [review]
It is bad practice to use ignore flags to avoid issues

@P1sar why I'm not set as a reviver on this PR?

**Comment by enemycnt** [comment]
@BeroBurny @P1sar In this PR I fixed linter errors only. The tests will be updated separately.

**Pull request merged**